src-ui/src/app/app-routing.module.ts

```diff
@@ -11,6 +11,7 @@ import { SettingsComponent } from './components/manage/settings/settings.compone
 import { TagListComponent } from './components/manage/tag-list/tag-list.component';
 import { NotFoundComponent } from './components/not-found/not-found.component';
 import { SearchComponent } from './components/search/search.component';
+import { DirtyFormGuard } from './guards/dirty-form.guard';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jonaswinkler/paperless-ng/pullRequest?prid=7592507)
```suggestion
import { DirtyFormGuard } from "./guards/dirty-form.guard";
```